ext/lib/mgmt/mcumgr/cmd/img_mgmt/src/img_mgmt.c

```diff
@@ -237,8 +237,14 @@ img_mgmt_erase(struct mgmt_ctxt *ctxt)
 {
     CborError err;
     int rc;
+    bool slot1_in_use;
```

Comment:
Initialize variables that are passed to functions via pointers; static analysis tools complain about this.